auction-server/build.rs

```diff
@@ -36,7 +36,6 @@ fn build_evm_contracts() {
     }
 }
 
-
```

Comment:
we can simplify this build script as well since we have committed the json files